tox.ini

```
@@ -101,13 +148,26 @@ setenv =
     remote_operator: PYTEST_PYTHON_FILES = test_remote_operator
     workflow: PYTEST_PYTHON_FILES = test_workflow
     service: PYTEST_PYTHON_FILES = test_service
+    api_entry: PYTEST_PYTHON_FILES = tests/entry
+    custom_type_field: PYTEST_PYTHON_FILES = test_custom_type_field
     operators: PYTEST_PYTHON_FILES = tests/operators
 
-deps = 
-    -r requirements/requirements_test.txt
+    TEMP = {env_tmp_dir}
+    TMP = {env_tmp_dir}
 
 commands =
-    pytest {env:PYTEST_PYTHON_FILES} {env:DEBUG} {env:COVERAGE} {env:RERUNS} {env:JUNITXML}
+    python -m pytest {env:PYTEST_PYTHON_FILES} {env:DEBUG} {env:RERUNS} {env:JUNITXML} {env:COVERAGE} {posargs}
+
+[testenv:report]
```

Feels too generic. What about?

```suggestion
[testenv:covreport]
```